build/Dockerfile.bundle

```diff
@@ -5,9 +5,9 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=oadp-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=stable
-LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.14.0
+LABEL operators.operatorframework.io.bundle.channels.v1=stable,stable-1.0
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable-1.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.18.0+git
```

Comment:
Unfortunately I don't know of a way to fix the `git` suffix, this happens with everyones installs on linux. So it's weird but fine by me. If you wanted ot manually remove it that works too.